src/uimports.cpp

```diff
@@ -106,6 +106,34 @@ static BOOL UPDATE_IMPORTS_XX(HANDLE hProcess,
         }
     }
 
+	///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+	//Added code
```

Comment:
Please remove these comments which won't make sense when the PR is merged.

Reply:
The latest commit has resolved it.

Reply:
Thanks!